torchio/transforms/augmentation/composition.py

```diff
@@ -20,12 +22,28 @@ class Compose(Transform):
     .. note::
         This is a thin wrapper of :py:class:`torchvision.transforms.Compose`.
     """
-    def __init__(self, transforms: Sequence[Transform], p: float = 1):
```

Comment:
Why is `transforms` this a kwarg now?

Reply:
I set a default value for `transforms` in order to keep the generic treatment of the reproducibility. In this implementation, the transform is initialized without any argument before setting the arguments manually using the `__dict__` attribute of the class. Since the `Compose` transform could not be initialized without arguments, I used this "trick" to not have a specific code for this class. I could try to find another way if you prefer